man/fi_domain.3.md

```diff
@@ -772,6 +773,11 @@ This specifies the default traffic class that will be associated any endpoints
 created within the domain.  See [`fi_endpoint`(3)](fi_endpoint.3.html)
 for additional information.
 
+## Max Authorization Keys per Endpoint (max_ep_auth_key)
+
+: The maximum number of authorization keys which can be supported per connectionless
+  endpoint.
```

Comment:
What is the correct behavior here if users specify zero? Following the precedence of similar fields, seems like if this value is zero, providers can return their default value?

Reply:
I think this depends on what a provider should do with this value if passed into an open domain/ep call.  Most apps will pass this value directly through.